package.json

```diff
@@ -76,15 +164,17 @@
     "time-grunt": "~1.2.2"
   },
   "engines": {
-    "node": ">=0.10.0"
+    "node": ">=0.10.0",
```

Comment:
I roughly encourage you to stick with what's Dedicated has already done:

https://github.com/ovh-ux/ovh-manager-dedicated/blob/master/package.json#L170-L173

```json
"engines": {
  "node": ">=8",
  "yarn": ">= 1.0.0"
},
```